.github/workflows/cypress-workflow.yml

```diff
@@ -7,9 +7,9 @@ on:
     branches:
       - "*"
 env:
-  OPENSEARCH_DASHBOARDS_VERSION: '2.5.0'
-  OPENSEARCH_VERSION: '2.5.0-SNAPSHOT'
-  ALERTING_PLUGIN_BRANCH: '2.5'
+  OPENSEARCH_DASHBOARDS_VERSION: '2.x'
```

Comment:
For context, the [OSD 2.6 branch](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/2.6) hasn't been cut yet, so I've configured the test workflows to use the `2.x` branch.